app/helpers/request_link_helper.rb

```diff
@@ -1,14 +1,10 @@
 module RequestLinkHelper
   def link_to_request_link(options = {})
     url = request_link(options[:document], options[:callnumber], options[:barcode])
+    link_text = 'Request'
+    link_text = 'Request on-site access' if Constants::REQUEST_ON_SITE_ACCESS_LIBS.include?(options[:library].try(:code))
     return unless url
```

Comment:
shouldn't `return unless url` be line 4?   And would an if / else be more obvious on assigning link_text?